### PR TITLE
SPECIES NUKING 2023: Refactors a bunch of species traits into flags for the head bodypart

### PR DESCRIPTION
**Pull request by ChungusGamer666**
## About The Pull Request

Title. 
I saw a comment on psyker code complaining about these species traits (which admittedly, they suck) and heeded the call to turn all of this crap into something defined almost entirely by the head bodypart.

## Why It's Good For The Game

Potential to simplify species code further in the future, by delegating more visuals to bodyparts, which is where most of them should be handled.

## Changelog

Should not be player facing, unless I fucked up.

### CODE REVIEW

**Comment by ChungusGamer666** [comment]
image
pref menu is OK
image
ingame humie is OK

**Comment by ChungusGamer666** [comment]
image
liggers are OK

thank you screenshot tests, time to fix the rest

**Comment by ChungusGamer666** [comment]
yeah i uhhh have no clue why a bunch of species get no eyes in the screenshots, i'll need some help man

**Comment by ChungusGamer666** [comment]
I FIGURED IT OUT!!!

**Comment by ChungusGamer666** [comment]
image
EVERYTHING FUCKING WORKS NOW, I HAD FUCKED UP THE ICON RENDER KEY LOL

**Comment by ChungusGamer666** [comment]
goodwillalwaystriumph

**Comment by ATH1909** [comment]
did you test HARS with this?

**Comment by ChungusGamer666** [comment]
> did you test HARS with this?

no, will test

**Comment by ChungusGamer666** [comment]
it will work with HARS

**Comment by ChungusGamer666** [comment]
head rendering code is still an absolute shitshow, but we have to start somewhere.
i'll try to keep working to make it not a shitshow after this is merged, if i did all i want to do at once this would be a nightmare for maintainerheads.

**Comment by ChungusGamer666** [comment]
image
lipstick looks like dogshit (due to human resprite, i assume), someone should resprite this blunder
but hey lipstick code works

**Comment by ChungusGamer666** [comment]
holy fuck lois im cumming

**Comment by ChungusGamer666** [comment]
yep
merge conflicts

**Comment by ChungusGamer666** [comment]
please mark this pr as a refactor, goofy bot is goofy

**Comment by ChungusGamer666** [comment]
this pr will actually kill me

**Comment by ChungusGamer666** [comment]
there is no merge conflict no more bitch

**Comment by ChungusGamer666** [comment]
can any maintainer label this properly as a refactor and remove the merge conflict label
k thanks

**Comment by ChungusGamer666** [comment]
@Time-Green you reviewed my previous species datum related pr, so this might interest you

**Comment by Time-Green** [comment]
> @Time-Green you reviewed my previous species datum related pr, so this might interest you

It do, I was just waiting for this to be finished. I’ll review it when I have some time

File: code/modules/religion/burdened/psyker.dm
Comment (Time-Green) [inline code/modules/religion/burdened/psyker.dm:48]: Nature is healing

File: code/modules/surgery/bodyparts/head.dm
Comment (Time-Green) [inline code/modules/surgery/bodyparts/head.dm:307]:
```suggestion
	return 
```
'.' is default, no need to specifiy

**Review by Time-Green — CHANGES_REQUESTED** [review]
Great work! Minor nitpicks/questions. Eyes, hair and lips should just be moved to bodypart_overlays, but that's for another day

File: code/modules/mob/living/carbon/human/_species.dm
Comment (Time-Green) [reply code/modules/mob/living/carbon/human/_species.dm:2314]: I'm gonna imagine you had an amazing counter argument

**Pull request merged**
